web/static/frontend/js/machine_page.js

```diff
@@ -1,11 +1,27 @@
 /**
- * Code to execute on machines page. Adds prompt for deleting machines.
+ * Code to execute on machines page.
+ * 
+ * - Enables clipboard.js for "Copy" buttons
+ * - Makes API key inputs select all text on click
+ * - Adds prompt for deleting machines
  */
 
+import Clipboard from "clipboard";
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)